File: ProjectShop.Server/Core/ObjectValue/JsonLogEntry.cs
namespace ProjectShop.Server.Core.ObjectValue
{
    public class JsonLogEntry
    {
        public DateTime QueryTime { get; set; } = DateTime.UtcNow;

        public object? EntityCall { get; set; }

        public object? Entity { get; set; }

        public string? ErrorName { get; set; }

        public string? Message { get; set; }

        public string? ErrorMessage { get; set; }
    }
}

File: projectshop.client/src/value-objects/JsonLogEntry.ts
export interface JsonLogEntry {
    queryTime: string;
    queryTimeString: string;
    entityCall: any | null;
    methodCall: any | null;
    entity: any | null;
    message: string | null;
    errorName: string | null;
    errorMessage: string | null;
    affectedRows: number | null;
}